tasks/travis-pr-title-lint.js

```diff
@@ -18,25 +18,53 @@ var writeCommitMessagesDocURL =
   '/CONTRIBUTING.md#writing-commit-messages';
 
 module.exports = function(grunt) {
-  function countGitMergeCommits() {
+  function findMergeBase() {
     return new Promise(function(resolve, reject) {
-      exec('git rev-list --count HEAD ^master', function(err, stdout) {
+      exec('git merge-base HEAD master', function(err, stdout) {
         if (err) {
           reject(err);
         } else {
-          resolve(parseInt(stdout));
+          var baseCommit = stdout.trim();
+          if (process.env.VERBOSE === 'true') {
+            console.log('DEBUG findMergeBase:', baseCommit); // eslint-disable-line no-console
+          }
+          resolve(baseCommit);
         }
       });
     });
   }
 
-  function getGitLastCommitMessage() {
+  function getGitBranchCommits() {
     return new Promise(function(resolve, reject) {
-      exec('git show -s --format=%B HEAD', function(err, stdout) {
+      findMergeBase().then(function(baseCommit) {
+        var gitCommand = 'git rev-list --no-merges HEAD ^' + baseCommit;
```

Comment:
By using `--no-merges` we are asking `git-rev-list` to do not list the merge commits.